prototype2/multigrid/mgmesytec/Efu.h

```diff
@@ -0,0 +1,37 @@
+/** Copyright (C) 2016-2018 European Spallation Source */
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// Class to parse detector readout for multigrid via
+/// sis3153 / Mesytec digitizer
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
Not sure I like the filename Efu.h which suggests something EFU generic but 
is located in a detector module.

Reply:
I don't like the name either. But it is short :) In a way, it is the essential neutron event formation step, as opposed to parsing or geometry resolution or any of those ancillary steps.
What it really should be is PositionDeterminationAlgorithm, but that's bad. Ideas?

Reply:
PositionAlgorithm 
PosAlgo
?